pyomo/contrib/doe/examples/fim_doe_tutorial.ipynb

```diff
@@ -57,77 +137,340 @@
     "\n",
-    "$A_1, A_2, E_1$, and $E_2$ are fitted model parameters. $R$ is the ideal-gas constant and $T$ is absolute temperature.\n",
+    "where:\n",
+    "* $A_1$ [$s^{-1}$], $A_2$ [$s^{-1}$] , $E_1$ [kJ/mol], and $E_2$ [kJ/mol] are fitted model parameters.\n",
+    "* $R$ [J/molK] is the ideal-gas constant.\n",
+    "* $T$ [K] is absolute temperature.\n",
     "\n",
-    "Using the **CCSI$^2$ toolset**, we would like to perform **uncertainty quantification** and **design of experiments** on a small-scale **batch reactor** to infer parameters $A_1$, $A_2$, $E_1$, and $E_2$."
+    "Using the Pyomo ecosystem, we would like to perform **uncertainty quantification** and **design of experiments** on a small-scale batch reactor to infer parameters $A_1$, $A_2$, $E_1$, and $E_2$."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Batch reactor\n",
+    "### Batch Reactor\n",
     "\n",
-    "The concentrations in a batch reactor evolve with time per the following differential equations:\n",
+    "The concentrations in a batch reactor evolves with time and are modeled by the following differential equations:\n",
     "\n",
     "$$ \\frac{d C_A}{dt} = r_A = -k_1 C_A $$\n",
     "\n",
     "$$ \\frac{d C_B}{dt} = r_B = k_1 C_A - k_2 C_B $$\n",
     "\n",
     "$$ \\frac{d C_C}{dt} = r_C = k_2 C_B $$\n",
     "\n",
-    "This is a linear system of differential equations. Assuming the feed is only species $A$, i.e., \n",
+    "We have now established a linear system of differential equations. Next, we can write the initial conditions assuming the feed is only species $A$ such that:\n",
     "\n",
-    "$$C_A(t=0) = C_{A0} \\quad C_B(t=0) = 0 \\quad C_C(t=0) = 0$$\n",
+    "$$C_A(t=0) = C_{A0}, \\quad C_B(t=0) = 0, \\quad C_C(t=0) = 0$$\n",
     "\n",
-    "When the temperature is constant, it leads to the following analytic solution:\n",
+    "When $k_1$ and $k_2$ are at constant temperature, it leads to the following analytic solution:\n",
     "\n",
-    "$$C_A(t) = C_{A,0} \\exp(-k_1 t)$$\n",
+    "$$C_A(t) = C_{A0} \\exp(-k_1 t)$$\n",
     "\n",
-    "$$C_B(t) = \\frac{k_1}{k_2 - k_1} C_{A,0} \\left[\\exp(-k_1 t) - \\exp(-k_2 t) \\right]$$\n",
+    "$$C_B(t) = \\frac{k_1}{k_2 - k_1} C_{A0} \\left[\\exp(-k_1 t) - \\exp(-k_2 t) \\right]$$\n",
     "\n",
-    "$$C_C(t) = C_{A,0} - \\frac{k_2}{k_2 - k_1} C_{A,0} \\exp(-k_1 t) + \\frac{k_1}{k_2 - k_1} \\exp(-k_2 t) C_{A,0} = C_{A,0} - C_{A}(t) - C_{B}(t)$$"
+    "$$C_C(t) = C_{A0} - \\frac{k_2}{k_2 - k_1} C_{A0} \\exp(-k_1 t) + \\frac{k_1}{k_2 - k_1} \\exp(-k_2 t) C_{A0} = C_{A0} - C_{A}(t) - C_{B}(t)$$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Implement Mathematical Model in Pyomo"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The mathematical model is comprised of a system of partial differential-algebraic equations (PDAEs) which will be solved using Pyomo.DAE."
```

Comment:
"comprised of a system of differential-algebraic equations (DAEs)", let's remove the "partial" for this notebook.